Reformat consistency error documentation (#8815)
mendix/docs: content/en/docs/refguide/modeling/consistency-errors/consistency-errors-navigation.md

## Navigation Consistency Errors 

The most common errors you can come across when configuring a navigation item are described in the following sub-sections.

### Error Code: CE1568

CE1568 error message: *The selected page {Name of the page} expects an object of type {type of object}, which is not available here.*

> You will get CE1568 if the page has a page parameter that expects an object of a particular type to be passed to it.

Reviewer: ```suggestion
You get CE1568 if the page has a page parameter that expects an object of a particular type to be passed to it.
```

    1. Change the **On click** property from **Show a page** to **Create object**.
    1. Set **ProgramItem** as **Entity (path)**. 
    1. Set **Program** as **On click page**. 
Now when an end-user clicks the menu item, a new *ProgramItem* object will be created and passed to the page.

### Error Code: CE0529

CE0529 error message: *The selected {Name of the page} expects an object of type {type of object} and cannot be used as a home page. Change the page or use a microflow to provide the page with an object.*

> You will get CE0529 if you have set a page that expects an object to be passed to it (for example, a page with a data view) as a home page. But the home page has no object that is passed to it, because it is the starting point of a flow.

Reviewer: ```suggestion
You get CE0529 if you have set a page that expects an object to be passed to it (for example, a page with a data view) as a home page. But the home page has no object that is passed to it, because it is the starting point of a flow.
```

> To fix CE0529, you can use a microflow as the home page that will open the preferred page and pass a specific object to the home page. For more information, see the [Error Fix Example for CE0529](#home-page-expects-an-object) section below.

Reviewer: ```suggestion
To fix CE0529, you can use a microflow as the home page that opens the preferred page and pass a specific object to the home page. For more information, see the [Error Fix Example for CE0529](#home-page-expects-an-object) section below.
```

{{< figure src="/attachments/refguide/modeling/consistency-errors/consistency-errors-navigation/open-home-page-microflow.png" alt="Open Home Page Microflow" class="no-border" >}}

### Error Code: CE0548

CE0548 error message: *Items with subitems cannot have an action themselves.*

> You will get CE0548 if you have assigned an [on-click event](/refguide/on-click-event/) to a menu item that has a sub-item, when menu items with have sub-items cannot have on-click events assigned to them.

Reviewer: ```suggestion
You get CE0548 if you have assigned an [on-click event](/refguide/on-click-event/) to a menu item that has a sub-item, when menu items with have sub-items cannot have on-click events assigned to them.
```
